kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiTHttpFrontendService.scala

```diff
@@ -274,15 +274,15 @@ final class KyuubiTHttpFrontendService(
     Option(SessionManager.getProxyHttpHeaderIpAddress).getOrElse(SessionManager.getIpAddress)
   }
 
-  override protected def getUserName(req: TOpenSessionReq): String = {
+  override protected def getUserName(req: TOpenSessionReq): (String, String) = {
     var userName: String = SessionManager.getUserName
     // using the remote ip address instead of that in proxy http header for authentication
     val ipAddress: String = SessionManager.getIpAddress
     if (userName == null) userName = req.getUsername
     userName = getShortName(userName)
     val effectiveClientUser: String = getProxyUser(req.getConfiguration, ipAddress, userName)
     debug("Client's username: " + effectiveClientUser)
-    effectiveClientUser
+    userName -> effectiveClientUser
```

Comment:
let's unify the terminologies

Reply:
Should these two users be reversed?
I think `effectiveClientUser` means `realUser` here, correct?

An alternative way is keeping this but reversing other places. (IMO, return `(super user, proxy user)` is more intuitive than `(proxy user, super user)`)